package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "markdown-link-check": "^3.7.3"
+  },
+  "scripts": {
+    "link-check": "find docs -name \\*.md -exec markdown-link-check {} \\;"
```

Comment:
```suggestion
    "link-check": "find docs -name \\*.md -print0 | xargs -0 -n1 markdown-link-check"
```

That will set exit code properly.

Reply:
Thanks for this suggestion, but on macOS it's not actually doing that. I want this to work for linux and OSX so kinda stuck with the hack approach. 

The local script doesnt need it so much, because its for humans. The CI does need it.

Reply:
<img width="696" alt="Screen Shot 2019-08-14 at 15 46 20" src="https://user-images.githubusercontent.com/7136002/63026023-b44a5980-beaa-11e9-84d6-812d4fb131e7.png">

<img width="728" alt="Screen Shot 2019-08-14 at 15 47 06" src="https://user-images.githubusercontent.com/7136002/63026071-ca581a00-beaa-11e9-8d7e-3cd2c9d85174.png">

Works perfectly on OSX ^

Reply:
<img width="599" alt="Screen Shot 2019-08-14 at 15 49 40" src="https://user-images.githubusercontent.com/7136002/63026549-aba65300-beab-11e9-925e-88b1c9647b31.png">
<img width="622" alt="Screen Shot 2019-08-14 at 15 49 29" src="https://user-images.githubusercontent.com/7136002/63026551-ac3ee980-beab-11e9-833a-cfecaf4e37ce.png">

OSX & zsh